bin/sculpin.php

```diff
@@ -54,7 +54,7 @@
 
     if (! is_dir($projectDir)) {
         throw new \InvalidArgumentException(
-            sprintf("Specified project directory %s does not exist", $projectDir)
+            sprintf('Specified project directory %s does not exist', $projectDir)
```

Comment:
Try and avoid changes like this next time, it muddies the water a little when reviewing.